sky/global_user_state.py

```diff
@@ -125,6 +128,9 @@ class ClusterStatus(enum.Enum):
     # Stopped.  This means a `sky stop` call has previously succeeded.
     STOPPED = 'STOPPED'
 
+    # Stopped.  This means a `sky down` call has previously succeeded.
+    TERMINATED = 'TERMINATED'
+
```

Comment:
We may not want to have the `TERMINATED` status in the enum as it will not actually appear in the `clusters` table.

How about we move the `colored_str` function to `cli_utils.status_utils.py` instead and change the `_STATUS_TO_COLOR` to:
```
_STATUS_TO_COLOR = {
    ClusterStatus.INIT.value: colorama.Fore.BLUE,
    ClusterStatus.UP.value: colorama.Fore.GREEN,
    ClusterStatus.STOPPED.vaue: colorama.Fore.YELLOW,
    'TERMINATED': <color for the terminated status>,
```